lib/Module/Build.pm

```diff
@@ -982,71 +981,54 @@ ExtUtils::MakeMaker> for further information.
-=back
-
-For years, these things have been a barrier to people getting the
-build/install process to do what they want.
+L<ExtUtils::MakeMaker> requires C<make> and use of a F<Makefile>.
+C<Module::Build> does not, nor do other pure-perl installers following the
+F<Build.PL> spec such as L<Module::Build::Tiny>. In practice, this is usually
+not an issue for the end user, as C<make> is already required to install most
+CPAN modules, even on Windows.
 
 =item *
 
-There are several architectural decisions in C<MakeMaker> that make it
-very difficult to customize its behavior.  For instance, when using
-C<MakeMaker> you do C<use ExtUtils::MakeMaker>, but the object created in
-C<WriteMakefile()> is actually blessed into a package name that's
-created on the fly, so you can't simply subclass
-C<ExtUtils::MakeMaker>.  There is a workaround C<MY> package that lets
-you override certain C<MakeMaker> methods, but only certain explicitly
-preselected (by C<MakeMaker>) methods can be overridden.  Also, the method
-of customization is very crude: you have to modify a string containing
-the Makefile text for the particular target.  Since these strings
-aren't documented, and I<can't> be documented (they take on different
-values depending on the platform, version of perl, version of
-C<MakeMaker>, etc.), you have no guarantee that your modifications will
-work on someone else's machine or after an upgrade of C<MakeMaker> or
-perl.
+L<ExtUtils::MakeMaker> has been a core module in every version of Perl 5, and
+must maintain compatibility to install the majority of CPAN modules.
+C<Module::Build> was added to core in Perl 5.10 and removed from core in Perl
+5.20, and (like L<ExtUtils::MakeMaker>) is only updated to fix critical issues
+and maintain compatibility. C<Module::Build> and other non-core installers like
+L<Module::Build::Tiny> are installed from CPAN by declaring themselves as a
+C<configure> phase prerequisite, and in this way any installer can be used in
+place of L<ExtUtils::MakeMaker>.
 
 =item *
 
-It is risky to make major changes to C<MakeMaker>, since it does so many
-things, is so important, and generally works.  C<Module::Build> is an
-entirely separate package so that I can work on it all I want, without
-worrying about backward compatibility with C<MakeMaker>.
+Customizing the build process with L<ExtUtils::MakeMaker> involves overriding
+certain methods that form the F<Makefile> by defining the subs in the C<MY::>
+namespace, requiring in-depth knowledge of F<Makefile>, but allowing targeted
+customization of the entire build. Customizing C<Module::Build> involves
+subclassing C<Module::Build> itself, adding or overriding pure-perl methods
+that represent build actions, which are invoked as arguments passed to the
+generated C<./Build> script. This is a simpler concept but requires redefining
+the standard build actions to invoke your customizations, and makes it more
+difficult to combine different customizations for the same action.
```

Comment:
In theory EUMM is more composable, problem is that there are probably about 10 people on this planet that know how to make it do the thing that was simple in M::I, making that point rather moot. In practice this hasn't been done on EUMM either, except through M::I.